opencontext/server/routes/vaults.py

```
@@ -361,69 +361,69 @@ async def trigger_document_processing(doc_id: int, document_data: dict, event_ty
+
         if result.get('success'):
-            logger.info(f"已清理文档 {doc_id} 的 {result.get('deleted_count', 0)} 个context chunks")
+            logger.info(f"Cleaned up {result.get('deleted_count', 0)} context chunks for document {doc_id}")
         else:
-            logger.warning(f"清理文档 {doc_id} 的context失败: {result.get('error')}")
-            
+            logger.warning(f"Failed to cleanup context for document {doc_id}: {result.get('error')}")
+
     except Exception as e:
-        logger.exception(f"清理文档context失败: {e}")
+        logger.exception(f"Failed to cleanup document context: {e}")
```

Ruff (0.13.3)

Redundant exception object included in `logging.exception` call

(TRY401)